### PR TITLE
Do final edits to proposal

### PR DESCRIPTION
**Pull request by KristinaRiemer**

### CODE REVIEW

File: proposal/problemdefinition.Rmd [hunk 1]
```diff
@@ -13,11 +13,17 @@ It is important to cover:
 
 
 Volatile organic compounds are important components of many biological and chemical processes, yet estimates of compound volatility are not available for most compounds. 
-Volatile organic compounds are chemicals that readily evaporate under ambient conditions, and are important in a number of fields and contexts including plant defense against herbivores, as routes of microbial communication, and as important indoor pollutants, to name a few.
-Yet measures of volatility are time consuming to calculate experimentally and not available for the vast majority of chemical compounds in chemical information databases. However, methods exist for predicting measures of volatility from chemical structure [@pankowSIMPOLSimpleGroup2008] that can be used to estimate compound volatility.
+These compounds are chemicals that readily evaporate under ambient conditions, and are important in a number of fields and contexts including plant defense against herbivores, as routes of microbial communication, and as important indoor pollutants, to name a few.
```
Comment (Aariq) [inline proposal/problemdefinition.Rmd]: Add "(VOC)" above as abbreviation for volatile organic compounds.  Then...
```suggestion
VOCs readily evaporate under ambient conditions and are important in a number of fields and contexts including plant defense, microbial communication, and indoor air pollution, to name a few.
```

File: proposal/problemdefinition.Rmd [hunk 2]
```diff
@@ -13,11 +13,17 @@ It is important to cover:
...
+Yet measures of volatility are time consuming to calculate experimentally, and volatility estimates are not available for the vast majority of chemical compounds in chemical information databases.
```
Comment (Aariq) [inline proposal/problemdefinition.Rmd]:
```suggestion
Yet measures of volatility are time consuming to calculate experimentally and not available for the vast majority of compounds in chemical information databases. 
```

File: proposal/problemdefinition.Rmd [hunk 3]
```diff
@@ -13,11 +13,17 @@ It is important to cover:
...
+Yet measures of volatility are time consuming to calculate experimentally, and volatility estimates are not available for the vast majority of chemical compounds in chemical information databases. 
 
-The `volcalc` package aims to automate the following steps for a given compound to estimate its volatility: 1) downloading data on chemical structure, 2) parsing those data to discover chemical functional groups, 3) applying the SIMPOL algorithm to predict volatility from functional groups and molecular weight, and assign compounds a volatility category given a reference environment
-The current draft version of `volcalc` does all this, but is limited to working with compounds present in the Kyoto Encyclopedia of Genes and Genomes (KEGG) database.
+The `volcalc` R package automates the process of estimating volatility of compounds. 
+It uses a method for predicting volatility from the chemical structure of the compound [@pankowSIMPOLSimpleGroup2008] to do so.
```
Comment (Aariq) [inline proposal/problemdefinition.Rmd]:
```suggestion
The `volcalc` R package automates the process of estimating volatility of compounds from their chemical structure [@pankowSIMPOLSimpleGroup2008].
```

File: proposal/problemdefinition.Rmd [hunk 4]
```diff
@@ -13,11 +13,17 @@ It is important to cover:
...
+Our package automates these following steps: 1) downloading data on chemical structure, 2) parsing those data to discover chemical functional groups, 3) applying the SIMPOL algorithm to predict volatility from functional groups and molecular weight, and assign compounds a volatility category given a reference environment.
```
Comment (Aariq) [inline proposal/problemdefinition.Rmd]:
```suggestion
Our package automates these following steps: 1) downloading data on chemical structure, 2) parsing those data to discover chemical functional groups, 3) applying the SIMPOL algorithm to predict volatility from functional groups and molecular weight, and assigning compounds a volatility category given a reference environment. 
```

File: proposal/problemdefinition.Rmd [hunk 5]
```diff
@@ -13,11 +13,17 @@ It is important to cover:
...
+The `volcalc` R package automates the process of estimating volatility of compounds. 
+It uses a method for predicting volatility from the chemical structure of the compound [@pankowSIMPOLSimpleGroup2008] to do so.
+Our package automates these following steps: 1) downloading data on chemical structure, 2) parsing those data to discover chemical functional groups, 3) applying the SIMPOL algorithm to predict volatility from functional groups and molecular weight, and assign compounds a volatility category given a reference environment. 
+This enables fast and easy estimation of volatility for thousands of chemical compounds. 
+
+While `volcalc` does this successfully, there are some limitations to its use that we would like to expand. 
+As the package currently is limited to working with compounds present in a single database and one type of file for describing compound structure, we would like to expand its use to additional compounds in more databases and more file types. 
+The package will also be more usable and reliable for researchers interested in volatility with an expansion of testing and documentation.
```
Comment (Aariq) [inline proposal/problemdefinition.Rmd]:
```suggestion
`volcalc` is currently limited to working with compounds present in a single database. We propose expanding its use to include any chemical with structural information—something widely available in many chemical information databases. In addition, we will improve testing and documentation to make the package more reliable.
```

File: proposal/problemdefinition.Rmd [hunk 6]
```diff
@@ -13,11 +13,17 @@ It is important to cover:
...
+Our package automates these following steps: 1) downloading data on chemical structure, 2) parsing those data to discover chemical functional groups, 3) applying the SIMPOL algorithm to predict volatility from functional groups and molecular weight, and assign compounds a volatility category given a reference environment. 
+This enables fast and easy estimation of volatility for thousands of chemical compounds. 
+
+While `volcalc` does this successfully, there are some limitations to its use that we would like to expand. 
+As the package currently is limited to working with compounds present in a single database and one type of file for describing compound structure, we would like to expand its use to additional compounds in more databases and more file types. 
+The package will also be more usable and reliable for researchers interested in volatility with an expansion of testing and documentation. 
 This proposal focuses on expanding the scope of `volcalc` and preparing it for a larger and more diverse user base so anyone interested in volatile organic compounds can integrate it into their workflow.
```
Comment (Aariq) [inline proposal/problemdefinition.Rmd]: repetitive of previous two sentences now.

File: proposal/proposal.Rmd [hunk 7]
```diff
@@ -10,9 +10,9 @@ At a high-level address what your proposal is and how it will address the proble
 
 
 Using the support from this proposal, we will take our existing tool for calculating volatility of compounds and expand its usage to makes predictions for essentially any compound, and enable researchers in a wide variety of fields to be able to use it with ease. 
-The current version of `volcalc` can estimate volatility only for compounds in one chemical database, called KEGG.
+The current version of `volcalc` can estimate volatility only for compounds in one chemical database, the Kyoto Encyclopedia of Genes and Genomes (KEGG).
 However, by refactoring existing code, we can make the volatility prediction functionality available to essentially any compound that has a known structure.
-Additionally, we plan to improve package infrastructure and documentation (i.e. tests, CI, vignettes, etc.), build the package on r-universe, and submit the package to CRAN in order to simplify installation.
+Additionally, we plan to improve package infrastructure and documentation (i.e. tests, CI, vignettes, etc.), build the package on r-universe, and submit the package to CRAN in order to simplify installation and make the package easier for users to find and use.
```
Comment (Aariq) [inline proposal/proposal.Rmd]:
```suggestion
Additionally, we plan to improve package infrastructure and documentation (i.e. tests, CI, vignettes, etc.), build the package on r-universe, and submit the package to CRAN in order to improve discoverability and simplify installation.
```

File: proposal/proposal.Rmd [hunk 8]
```diff
@@ -27,14 +27,17 @@ Depending on project type the detail section should include:
 
 The current version of `volcalc` was created in 2022 as part of a [data science incubator](https://datascience.cct.arizona.edu/cct-data-science-incubator) project in collaboration between Dr. Kristina Riemer and Dr. Laura Meredith at University of Arizona.
 `volcalc` is the first project, to our knowledge, to implement the SIMPOL method for predicting chemical vapor pressures and enthalpies of vaporization [@pankowSIMPOLSimpleGroup2008] in an R package.
-This current version of the `volcalc` package has been successfully used to calculate volatility estimates for *all* 16,000+ compounds in the Kyoto Encyclopedia of Genes and Genomes (KEGG) database (in prep ms ref). validated using hand-collected data [@meredith2022; @honeker2021]
+This current version of the `volcalc` package has been successfully used to calculate volatility estimates for *all* 16,000+ compounds in the KEGG database [@meredith].
```
Comment (Aariq) [inline proposal/proposal.Rmd]:
```suggestion
volcalc` has been successfully used to calculate volatility estimates for *all* 16,000+ compounds in the KEGG database [@meredith]. 
```

File: proposal/proposal.Rmd [hunk 9]
```diff
@@ -27,14 +27,17 @@ Depending on project type the detail section should include:
...
-This current version of the `volcalc` package has been successfully used to calculate volatility estimates for *all* 16,000+ compounds in the Kyoto Encyclopedia of Genes and Genomes (KEGG) database (in prep ms ref). validated using hand-collected data [@meredith2022; @honeker2021]
+This current version of the `volcalc` package has been successfully used to calculate volatility estimates for *all* 16,000+ compounds in the KEGG database [@meredith]. 
+To ensure accuracy of the package, the volatility estimates for a subset of the compounds were validated using hand-collected data [@meredith2022; @honeker2021]. 
 
-Because `volcalc` is currently limited to only working with chemical compounds from KEGG, we will refactor it to take molfiles as inputs directly.
+Because `volcalc` is currently limited to only working with chemical compounds from KEGG, we will refactor it to take chemical compound structure files as inputs directly.
+The type of structure file the packagee uses is called a molfile.
```
Comment (Aariq) [inline proposal/proposal.Rmd]: I think this has already been said and can maybe be removed?
```suggestion

```

File: proposal/proposal.Rmd [hunk 10]
```diff
@@ -27,14 +27,17 @@ Depending on project type the detail section should include:
...
+The type of structure file the packagee uses is called a molfile. 
 The main function in `volcalc`, `calc_vol()`, currently downloads chemical information as molfiles from the KEGG API as a starting point.
-It reads in that .mol file and parses it to find functional groups.
+It reads in that molfile and parses it to find functional groups.
 It finally applies an algorithm (published in @pankowSIMPOLSimpleGroup2008) to predict vapor pressure, and outputs to the user a relative measure of volatility.
 
-Molfile is an open format and various tools exist to translate other standard representations of chemical structure such as SMILES [@weininger1988smiles] and InChI [@heller2013] to molfiles (e.g. using the [`OpenBabel`](http://openbabel.org/wiki/Main_Page) command line utility or in R with [`ChemmineOB`](https://github.com/girke-lab/ChemmineOB)).
+A molfile is an open format, and various tools exist to translate other standard representations of chemical structure such as SMILES [@weininger1988smiles] and InChI [@heller2013] to molfiles.
```
Comment (Aariq) [inline proposal/proposal.Rmd]: I think it was correct before.  Like saying "CSV is an open format" or "XLSX is a proprietary format".
```suggestion
Molfile is an open file format, and various tools exist to translate other standard representations of chemical structure such as SMILES [@weininger1988smiles] and InChI [@heller2013] to molfiles. 
```

File: proposal/proposal.Rmd [hunk 11]
```diff
@@ -27,14 +27,17 @@ Depending on project type the detail section should include:
...
 It finally applies an algorithm (published in @pankowSIMPOLSimpleGroup2008) to predict vapor pressure, and outputs to the user a relative measure of volatility.
 
-Molfile is an open format and various tools exist to translate other standard representations of chemical structure such as SMILES [@weininger1988smiles] and InChI [@heller2013] to molfiles (e.g. using the [`OpenBabel`](http://openbabel.org/wiki/Main_Page) command line utility or in R with [`ChemmineOB`](https://github.com/girke-lab/ChemmineOB)).
+A molfile is an open format, and various tools exist to translate other standard representations of chemical structure such as SMILES [@weininger1988smiles] and InChI [@heller2013] to molfiles. 
+This translation can be done using the [`OpenBabel`](http://openbabel.org/wiki/Main_Page) command line utility or in R with [`ChemmineOB`](https://github.com/girke-lab/ChemmineOB).
 Since `ChemmineOB` is already a dependency of `volcalc`, we can additionally extend `volcalc` to allow other representations of chemical structures as inputs.
 Since SMILES and InChl are both string representations, this will ideally make `volcalc` fit more easily into a `data.frame`-based workflow.
```
Comment (Aariq) [inline proposal/proposal.Rmd]: Shorten 
```suggestion
This translation can be done using  [`ChemmineOB`](https://github.com/girke-lab/ChemmineOB) which is already a dependency of `volcalc`. SMILES and InChl are both string representations, so adding these as possible inputs will make `volcalc` fit more easily into a `data.frame`-based workflow.
```

File: proposal/success.Rmd [hunk 12]
```diff
@@ -18,20 +18,23 @@ The following can be used to measure success:
...
-- The number of chemical representation types that can be used as input (molfile for success; inchikey, inchi, SMILES, and possibly more for reach goal)
+- Can chemical representation types be used as input? (molfile for success; InChI, SMILES, and possibly more for reach goal)
 
 ## Future work
 <!-- 
 How could this be extended / developed in the future by yourself and/or the community in general?
 -->
 
-**TODO: replace with text from future directions in manuscript in prep**
+- As a handful of functional groups relevant to volatility were not able to be measured using existing methods, the community who creates these methods could expand to these groups
```
Comment (Aariq) [inline proposal/success.Rmd]: Feel free to reject suggestion if I'm not getting your meaning.

```suggestion
- The contribution of some functional groups to volatility were not able to be measured using existing methods. As these analytical methods improve, the algorithm in `volcalc` can be updated.
```

File: proposal/success.Rmd [hunk 13]
```diff
@@ -18,20 +18,23 @@ The following can be used to measure success:
...
+- The algorithm used in the package only calculates relative volatility under standard conditions; the algorithm could be added to or expanded to calculate volatility itself and for additional environmental conditions
```
Comment (Aariq) [inline proposal/success.Rmd]:
```suggestion
- `volcalc` only calculates relative volatility under standard conditions; functionality to specify environmental conditions could be added.
```

File: proposal/success.Rmd [hunk 14]
```diff
@@ -18,20 +18,23 @@ The following can be used to measure success:
...
+- As additional chemical compound databases are used as inputs to `volcalc`, these should be documented and either added to this package or created as standalone packages that link to `volcalc`
```
Comment (Aariq) [inline proposal/success.Rmd]: I would just remove this.  I don't think we want to get into the API wrapper game
```suggestion
```

File: proposal/success.Rmd [hunk 15]
```diff
@@ -18,20 +18,23 @@ The following can be used to measure success:
...
+- As this package is used increasingly by researchers, they will discover other edge cases or unusual compounds that the package can be improved to accomodate
```
Comment (Aariq) [inline proposal/success.Rmd]: I don't think this is necessary here.  Could be part of the proposal somewhere else to mention that the Meredith lab and CCT will continue maintainance and development after this grant instead.
```suggestion
```

File: proposal/timeline.Rmd [hunk 16]
```diff
@@ -19,7 +19,8 @@ Covering the planning phase, this section should provide a relatively detailed p
 
 - Implement CI with GitHub actions
 - Check code coverage with `codecov` package
-- Use GitHub Issues or Discussions to brainstorm eventual API (i.e. function names, argument names, how many exported functions, etc.)
+- Implement user inputs for environment (soil, water) and volatility categories
```
Comment (Aariq) [inline proposal/timeline.Rmd]: Can we move this to milestone 2?  I see milestone 1 as just getting infrastructure set up and making a plan

**Review by Aariq — CHANGES_REQUESTED** [review]
Tried to shorten things a little bit and provide other feedback.

File: references.bib [hunk 17]
```diff
@@ -77,3 +77,9 @@ @article{honeker2021
 	doi = {10.3389/fenvs.2021.649905},
 	url = {http://dx.doi.org/10.3389/fenvs.2021.649905}
 }
+
+@article{meredith,
+	title = {Automating methods for estimating metabolite volatility},
+	author = {Meredith, L.K. and Riemer, K. and Geffre, P. and Honeker, L. and Krechmer, J. and Graves, K. and Tfaily, M. and Ledford, S.K.},
+	year = {in prep}
```
Comment (Aariq) [inline references.bib]: Make "in prep" itallics:
```suggestion
	year = {\emph{in prep}}
```

File: proposal/timeline.Rmd (hunk 16 above)
Comment (KristinaRiemer) [reply proposal/timeline.Rmd]: Done in 4b1213c

File: proposal/problemdefinition.Rmd (hunk 6 above)
Comment (KristinaRiemer) [reply proposal/problemdefinition.Rmd]: Removed in 4b1213c

**Pull request merged**